jvm/src/main/java/com/intel/oap/vectorized/CHJniInstance.java

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
Can we restructure the files? For example, the row related files can be put into a specific row folder, not in the **vectorized** foler. Also, the CH related codes can be reused by other libs as well, so I think their names and paths should not contain library related information.

Reply:
> Can we restructure the files? For example, the row related files can be put into a specific row folder, not in the **vectorized** foler. Also, the CH related codes can be reused by other libs as well, so I think their names and paths should not contain library related information.

Sure, we can restructure these files in the following PR, when JNI interfaces are consistent, this file isn't needed and will be removed.

Reply:
Done